### PR TITLE
Named exports

### PR DESCRIPTION
**Pull request by TylorS**
Avoids oddities with module.exports and exports.default
Unifies TypeScript definitions that support both ES6 and ES5

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,4 +4,6 @@ import { Stream } from 'most'
 import { MulticastSource } from '@most/multicast'
 import Create from './Create'
 
-export default run => new Stream(new MulticastSource(new Create(run)))
+export function create (run) {
+  return new Stream(new MulticastSource(new Create(run)))
```
Comment (davidchase) [inline src/index.js:8]: question whats the difference between that and this ?

``` js
export const create = run => new Stream(new MulticastSource(new Create(run)))
```

Comment (TylorS) [reply src/index.js:8]: Just personal preference, really.
I can adjust it if that'd be preferred.

Comment (davidchase) [reply src/index.js:8]: ah ok.. gotcha didnt know if there was something different that i missed

**Comment by davidchase** [comment]
i think this is good to go 👍

**Pull request merged**